units/src/amount/mod.rs

```diff
@@ -109,6 +112,7 @@ impl Denomination {
             Denomination::MicroBitcoin => -2,
             Denomination::Bit => -2,
             Denomination::Satoshi => 0,
+            Denomination::_Never(_) => unreachable!("cannot create this variant"),
```

Comment:
This can be statically proven and it should be.

Reply:
I couldn't get the `match {}` thing to build, I should have said that when pushing.

Reply:
That's impossible, we've used that idiom multiple times in the crate. What's the error message?

Reply:
Agh, thanks!